src/app/components/timeline/BlankCard.js

```diff
@@ -6,119 +6,225 @@ import { bindActionCreators } from 'redux'
+    this.props.actions.addCard(
+      Object.assign(newCard, this.state.templates ? { templates: this.state.templates } : {})
+    )
+    this.setState({
+      creating: false,
+      templates: [],
+    })
+    if (this.props.onDone) this.props.onDone()
   }
 
   handleFinishCreate = (event) => {
-    if (event.which === 13) { //enter
+    if (event.which === 13) {
+      //enter
       this.saveCreate()
     }
   }
 
   startCreating = () => {
-    this.setState({creating: true})
+    this.setState({ creating: true })
   }
 
   buildCard (title) {
     const { chapterId, lineId } = this.props
     if (this.props.isSeries) {
-      return { title, beatId: chapterId, seriesLineId: lineId, positionWithinLine: 0 }
+      return {
+        title,
+        beatId: chapterId,
+        seriesLineId: lineId,
+        positionWithinLine: this.props.positionWithinLine || 0,
+      }
     } else {
-      return { title, chapterId, lineId, positionWithinLine: 0 }
+      return { title, chapterId, lineId, positionWithinLine: this.props.positionWithinLine || 0 }
     }
   }
 
   handleCancelCreate = (event) => {
-    if (event.which === 27) { //esc
-      this.setState({creating: false})
+    if (event.which === 27) {
+      //esc
+      this.setState({ creating: false })
     }
   }
 
   handleBlur = () => {
     var newTitle = findDOMNode(this.refs.titleInput).value
     if (newTitle === '') {
-      this.setState({creating: false})
+      this.setState({ creating: false })
       return false
     } else {
       this.saveCreate()
-      this.setState({creating: false})
+      this.setState({ creating: false })
     }
   }
 
+  onAddWithTemplateHover = () => {
+    this.setState({
+      templateHover: true,
+    })
+  }
+
+  onAddWithTemplateLeave = () => {
+    this.setState({
+      templateHover: false,
+    })
+  }
+
+  onAddWithDefaultHover = () => {
+    this.setState({
+      defaultHover: true,
+    })
+  }
+
+  onAddWithDefaultLeave = () => {
+    this.setState({
+      defaultHover: false,
+    })
+  }
+
+  showTemplatePicker = () => {
+    this.setState({
+      showTemplatePicker: true,
+    })
+  }
+
+  handleChooseTemplate = (template) => {
+    const newTemplates = this.state.templates.find(({ id }) => id === template.id)
+      ? this.state.templates
+      : [template, ...this.state.templates]
+
+    this.setState({
+      templates: newTemplates,
+      showTemplatePicker: false,
+      creating: true,
+    })
+  }
+
+  closeTemplatePicker = () => {
+    this.setState({ showTemplatePicker: false })
+  }
+
   renderBlank () {
-    var blankCardStyle = {
-      borderColor: this.props.color
+    const blankCardStyle = {
+      borderColor: this.props.color,
+      color: this.props.color,
     }
-    return <div className={cx('blank-card__body', {hover: this.state.dropping})} style={blankCardStyle} />
+    const addWithTemplateStyle = this.state.templateHover
+      ? {
+          background: this.props.backgroundColor,
```

Comment:
couldn't this just be `this.props.color`?

Reply:
It doesn't look good just using colour for the background colour.
It's way too strong for the text to come through and stands out too much on the UI.